src/providers/bttv/BttvEmotes.cpp

```diff
@@ -94,7 +94,7 @@ namespace {
                     ImageSet{
                         Image::fromUrl(getEmoteLinkV3(id, "1x"), 1),
                         Image::fromUrl(getEmoteLinkV3(id, "2x"), 0.5),
-                        Image::fromUrl(getEmoteLinkV3(id, "3x"), 0.25),
+                        Image::fromUrl(getEmoteLinkV3(id, "3x"), 0.286),
```

Comment:
The BTTV 3x emote is actually 4x-scaled of the 1x-version

Take KKona for example
```sh
for size in {1,2,3}; do echo "$size"; curl -s https://cdn.betterttv.net/emote/566ca04265dbbdab32ec054a/${size}x | file -; done
1
/dev/stdin: PNG image data, 25 x 34, 8-bit/color RGBA, non-interlaced
2
/dev/stdin: PNG image data, 50 x 68, 8-bit/color RGBA, non-interlaced
3
/dev/stdin: PNG image data, 100 x 136, 8-bit/color RGBA, non-interlaced
```

Then KKoooona from FFZ
```sh
for size in {1,2,4}; do echo "$size"; curl -s https://cdn.frankerfacez.com/emoticon/164405/$size | file -; done
1
/dev/stdin: PNG image data, 80 x 20, 8-bit/color RGBA, non-interlaced
2
/dev/stdin: PNG image data, 160 x 40, 8-bit/color RGBA, non-interlaced
4
/dev/stdin: PNG image data, 320 x 80, 8-bit/color RGBA, non-interlaced
```

The ratios are the same yet they're called differently


7TV's supports all 4 versions and the scales are all correct
```sh
for size in {1,2,3,4}; do echo "$size"; curl -s https://cdn.7tv.app/emote/603cac391cd55c0014d989be/${size}x.png | file -; done
1
/dev/stdin: PNG image data, 42 x 32, 8-bit/color RGBA, non-interlaced
2
/dev/stdin: PNG image data, 84 x 64, 8-bit/color RGBA, non-interlaced
3
/dev/stdin: PNG image data, 126 x 96, 8-bit/color RGBA, non-interlaced
4
/dev/stdin: PNG image data, 168 x 128, 8-bit/color RGBA, non-interlaced
```

I believe the correct change here would be to omit the 3x version, and treat BTTV's 3x version as a 4x version